locales/base/translation.json

```
@@ -2694,6 +2694,25 @@
         "yieldRateTitle": "Yield Rate",
         "yieldRateDescription": "While most pools offer earnings in the form of a liquidity pool token, some give additional token(s) as a reward or added incentive.\n\nSince {{appName}} aggregates pools across multiple protocols, we have combined all the earning and reward rates into a single, overall yield rate to help easily evaluate your earning potential. This number is an estimate since the earning and reward values fluctuate constantly.\n\nFor further information about earning breakdowns you can visit <0>{{providerName}}</0>."
       }
+    },
+    "beforeDepositBottomSheet": {
+      "youNeedTitle": "You Need {{tokenSymbol}} on {{tokenNetwork}} to Deposit",
+      "crossChainAlternativeDescription": "If you don’t want to use your tokens on {{tokenNetwork}}, choose an option below. You’ll need to return to complete your pool deposit later.",
+      "beforeYouCanDepositTitle": "Before you can deposit...",
+      "beforeYouCanDepositDescription": "You’ll need to add one of the pool tokens. Once added, you’ll need to return to complete your pool deposit.",
+      "swapActionDescription": "Swap into {{tokenSymbol}} on {{tokenNetwork}} from another token in your wallet",
+      "action": {
+        "swapAndDeposit": "Swap & Deposit",
+        "swapAndDepositActionDescription": "Choose any token on {{tokenNetwork}}. We’ll swap and deposit it simultaneously for you.",
```

```suggestion
        "swapAndDepositDescription": "Choose any token on {{tokenNetwork}}. We’ll swap and deposit it simultaneously for you.",
```
nit: drop the action from these as there's already an `action` one level above. You could also nest actions. E.g., `<action-name>: { 'title': '...', 'description' }`, but seems like other sections of these don't do it, so not strictly required